web-wallet/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Update Tabs to use native scroll behavior [#1320]
```

Comment:
After the discussion we had with Matteo is this change something that needs to be shown in the changelog?

Reply:
I think so, as the user can see that the scrolling behavior has actually changed (as in, it's a user-facing change).

Reply:
(although there are no scrolling tabs in our application)

Reply:
Haha that's true too. I will leave it, I don't think it matters much in this case.